libs/product/src/drivers/magento/transforms/bundled-product-transformers.ts

```diff
@@ -0,0 +1,40 @@
+import { MagentoBundledProduct, MagentoBundledProductItem, MagentoBundledProductItemOption } from '../models/bundled-product';
+import { DaffProductTypeEnum } from '../../../models/product';
+import { DaffCompositeProduct } from '../../../models/composite-product';
+import { 
+	DaffCompositeProductItemOption, 
+	DaffCompositeProductItem, 
+	DaffCompositeProductItemInputEnum 
+} from '../../../models/composite-product-item';
+import { transformMagentoSimpleProduct } from './simple-product-transformers';
+
+/**
+ * Transforms the magento MagentoProduct from the magento product query into a DaffProduct. 
+ * @param response the response from a magento product query.
+ */
+export function transformMagentoBundledProduct(product: MagentoBundledProduct, mediaUrl: string): DaffCompositeProduct {
```

Comment:
I think this transformer should be exported in `public_api.ts`.

Reply:
Done. I'd be surprised if anyone ever uses it outside of daffodil though.

Reply:
reverted back per Damien's comment